### PR TITLE
Add command for rerunning the latest project workflow on CircleCI [PREMIUM-207]

### PR DESCRIPTION
**Pull request by lysyjan**
## Description

When we release premium and free plugins together, our first builds for the premium plugin on Circle CI fail. This PR adds a command that reruns the latest Circle CI workflow for a specific project. The new command is also used in the prepare command.

## Code review notes

_N/A_

## QA notes

_N/A_

## Linked PRs

_N/A_

## Linked tickets

[PREMIUM-207](https://mailpoet.atlassian.net/browse/PREMIUM-207)

## After-merge notes

_N/A_

### CODE REVIEW

File: mailpoet/RoboFile.php
Comment (rodrigoprimo) [inline mailpoet/RoboFile.php]: I guess we need to put this line inside an else condition or finish the execution of this method when yell() is called to avoid having both the error and the success message being printed when there is an error?

**Review by rodrigoprimo — COMMENTED** [review]
Overall the code changes look good to me, @lysyjan. I left a comment with a small detail that I believe we should fix before merging this. Besides that, I wonder if there is a way to test the changes in this PR before I approve it? IMO, it would be good if we could test this before using it in the next release next week.

File: mailpoet/RoboFile.php
Comment (lysyjan) [reply mailpoet/RoboFile.php]: Good catch! I fixed it in the latest commit.

**Comment by lysyjan** [comment]
@rodrigoprimo You can test the new command by following these steps:
1. Change the branch name on [this line](https://github.com/mailpoet/mailpoet/pull/4537/files#diff-7fd98d4fe52b90d96a196f1ff8179ad830d024eb35e06f3e14b18e51a1116b53R171)
2. Execute command `./do release:rerun-circle-workflow`.

I have noticed now that I haven't added the command to the premium repository. At this moment, we don't need it, but it might be nice to have the release commands consistent. What do you think?

File: mailpoet/RoboFile.php
Comment (rodrigoprimo) [reply mailpoet/RoboFile.php]: Thanks

**Comment by rodrigoprimo** [comment]
> You can test the new command by following these steps

Thanks for the testing instructions. I was able to force CircleCI to restart the build for a test branch called `delme`. The commands seem to be working as expected.

> I have noticed now that I haven't added the command to the premium repository. At this moment, we don't need it, but it might be nice to have the release commands consistent. What do you think?

I don't have a strong opinion here, but I'm inclined to think that if we don't need it at this moment, we shouldn't include it. We can always include it later if needed. I'm also unsure which build job we would want to rerun when running `./do release:prepare` for the premium plugin. Let me know if I'm missing something. I will leave this PR as approved. Feel free to move it to QA if you think that is needed (probably not), merge it, or continue the conversation about adding the command to the premium repository.

**Review by rodrigoprimo — APPROVED** [review]

**Comment by lysyjan** [comment]
@rodrigoprimo Thank you. I'm okay with the current state, and I will merge it.

**Pull request merged**
